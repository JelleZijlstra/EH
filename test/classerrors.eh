#!/usr/bin/ehi
class Foo {
	const toString = func:
		'@Foo'
	end

	try
		for 5 count i
			const foo = i
		end
	catch
		echo exception.toString()
	end
	public mah = 0
	for 10 count i
		mah++
	end
	echo mah
}
