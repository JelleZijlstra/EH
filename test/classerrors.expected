%%stdout%%
<ConstError: Cannot set constant member foo in object of type Class: (class <Foo>)>
10
%%stderr%%
